static/templates/template/container.html

```diff
@@ -1,8 +1,8 @@
-<md-subheader class="md-no-sticky">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
+<md-subheader class="md-no-sticky" ng-init="project.getLastMilestoneComments()">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
```

Comment:
We do not want to keep calling this API every time tab changes, so it was called using ng-init only when tab - Design is loaded.